### PR TITLE
Add DeleteForwardUrls to master

### PR DESCRIPTION
**Pull request by awong503**
1. delete from forward_urls table
2. add schema - deleteforwardurl mutation for graphql
3. used to dropdown "selection" piece on webhook-buddy-client

### CODE REVIEW

File: src/models/forwardUrl.ts [hunk 1]
```diff
@@ -82,3 +82,27 @@ const prune = async (
...
+    f.endpoint_id = $1
+    AND
+    f.url = $2
+    AND
+    EXISTS (
+      SELECT 1
+      FROM endpoints as e
+      INNER JOIN user_endpoints as ue on ue.endpoint_id = e.id
+      WHERE e.id = f.endpoint_id AND ue.user_id = $3
+    )`,
```
Comment (johnnyoshika) [inline src/models/forwardUrl.ts]: This `EXISTS` seems unnecessarily complex. I believe you got inspiration from here: https://github.com/webhook-buddy/webhook-buddy-server/blob/master/src/models/webhook.ts#L167-L172

But in that sample code, the extra EXISTS is an extra layer of protection to ensure that someone isn't deleting a webshook to which s/he doesn't have access. In your case, the user_id is in the forward_urls table, so we can just check for that. So something like this should do the trick:

```
AND
f.user_id = $3
```

**Review by johnnyoshika — CHANGES_REQUESTED** [review]
@awong503 Looking good. One small change and it should be ready to merge. Thanks!

File: src/models/forwardUrl.ts (hunk 1 above)
Comment (awong503) [reply src/models/forwardUrl.ts]: Thanks for clarifying! I have corrected and pushed. Tested in graphql playground and whb-client, it still works.

Comment (awong503) [reply src/models/forwardUrl.ts]: @johnnyoshika

**Review by johnnyoshika — APPROVED** [review]

**Pull request merged**